backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2522,6 +2525,38 @@ bool SurfaceSdlGraphicsManager::notifyEvent(const Common::Event &event) {
 			return true;
 		}
 
+#ifdef WIN32
+		// Alt-S: Create a screenshot - Win32
+		if (event.kbd.hasFlags(Common::KBD_ALT) && event.kbd.keycode == 's') {
+			Common::String filename;
+
+			Common::String screenshotsPath = OSystem_Win32::getScreenshotsPath();
+
+			char fullpath[MAXPATHLEN];
+
+			for (int n = 0;; n++) {
+				SDL_RWops *file;
+
+				filename = Common::String::format("scummvm%05d.bmp", n);
+
+				strcpy(fullpath, screenshotsPath.c_str());
+				strcat(fullpath, filename.c_str());
+
+				file = SDL_RWFromFile(fullpath, "r");
+
+				if (!file)
+					break;
+				SDL_RWclose(file);
+			}
+
+			if (saveScreenshot(fullpath))
+				debug("Saved screenshot '%s'", filename.c_str());
+			else
+				warning("Could not save screenshot");
+
+			return true;
+		}
+#else
```

Comment:
Again, you can factorise a lot of code between the windows case and the else case.